### PR TITLE
Create android_do

### PR DESCRIPTION
**Pull request by sssemil**

### CODE REVIEW

File: android_do [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    }
+    [[ "$cpu_arch" = "i686" ]] \
+        && cpu_arch="x86"
+
+    NDK=$BUILD_PATH/$NDK_VERSION/
+
+    ##SETUP NDK
+    cd "$BUILD_PATH"
+    if [ ! -d "$NDK" ]; then
+          echo "$NDK_VERSION-linux-${cpu_arch}.tar.bz2"
```
Comment (cjdelisle) [inline android_do]: It would be quite nice if this could work on mac too...

File: android_do [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        echo "The NDK variable is not pointing to a valid directory"
+        exit 1
+    }  
+    cd ..
+  else
+    NDK=$BUILD_PATH/$NDK_VERSION/
+  fi
+fi
+
+$NDK/build/tools/make-standalone-toolchain.sh --platform=android-21 --toolchain=arm-linux-androideabi-4.9 --install-dir=$BUILD_PATH/arm-linux-androideabi/ --system=linux-x86_64
```
Comment (cjdelisle) [inline android_do]: Does this work on linux-x86 ?

**Comment by sssemil** [comment]
I don't have mac support needs to be checked.

**Comment by cjdelisle** [comment]
sorry for the delay, pulling this now and will test it tomorrow when I have a good internet connection.

**Comment by cjdelisle** [comment]
merged to crashey, will release in versions 16.4 and 17, thanks

**Pull request closed without merging**